go.mod

```diff
@@ -5,10 +5,10 @@ go 1.23
 require (
 	github.com/eclipse/paho.mqtt.golang v1.5.0
 	github.com/edgexfoundry/go-mod-bootstrap/v4 v4.0.0-dev.1
-	github.com/edgexfoundry/go-mod-configuration/v4 v4.0.0-dev.1
+	github.com/edgexfoundry/go-mod-configuration/v4 v4.0.0-dev.2
```

Comment:
we should remove the Consul dependency from go-mod-configuration and go-mod-registry first, and we can make sure the services could still be built and run without the consul libraries